src/core/assertions.ts

```diff
@@ -1,16 +1,8 @@
-'use strict'
-
-/**
- * @module Assertions
- */
-
-const _ = require('lodash')
-const { expect, use } = require('chai')
-const moment = require('moment-timezone')
-const Cast = require('./cast')
-const { registerChaiAssertion } = require('./custom_chai_assertions')
-
-use(registerChaiAssertion)
+import * as assert from 'node:assert/strict'
```

Comment:
Great idea using node's assert instead of chai's expect, we don't need the later.